<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDimensionParameters</class>
 <widget class="QWidget" name="CDimensionParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDimensionParameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../DyssolMainWindow/Resources.qrc">
    <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxGridEntry"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Classes:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Entry:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBoxUnits"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDistrFunction">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxGridFun"/>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBoxClasses">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelUnits">
       <property name="text">
        <string>Units:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelMax">
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelMin">
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditLimitMin">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEditLimitMax">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CQtTable" name="tableWidgetGrid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDispUnits">
       <property name="text">
        <string>Units to display:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDispUnits"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtTable</class>
   <extends>QTableWidget</extends>
   <header location="global">QtTable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxGridEntry</tabstop>
  <tabstop>spinBoxClasses</tabstop>
  <tabstop>comboBoxGridFun</tabstop>
  <tabstop>comboBoxUnits</tabstop>
  <tabstop>lineEditLimitMin</tabstop>
  <tabstop>lineEditLimitMax</tabstop>
  <tabstop>tableWidgetGrid</tabstop>
  <tabstop>comboBoxDispUnits</tabstop>
 </tabstops>
 <resources>
  <include location="../../DyssolMainWindow/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
